lib/conn-pool.js

```diff
@@ -1,4 +1,3 @@
-const arrayRemove = require('unordered-array-remove')
```

Comment:
Could `unordered-array-remove` dep be removed or are the still places where it's used?

Reply:
if you mean this specific file, then it's not used, but unordered-array-remove is still used in other files such as [peer.js](https://github.com/webtorrent/webtorrent/blob/master/lib/peer.js#L272)

Reply:
I think i only touched the internals (no public facing properties) that would mean any breaking changes. that's why i think i left out peer.js perhaps.